Index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="Style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,500;0,600;0,700;1,400;1,500;1,600;1,700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <nav class="navbar">
+    <h2 class="myLogo">My logo</h2>
+    <img class="burger" src="Assests/Union.jpg">
```

Comment:
- [ ] The hamburger button should have a `nav` tag. The `nav` tag is for semantics reasons. In other words, your logo should be placed outside your `nav` tag, for semantic reasons. You should only place your navbar links and hamburger/menu icon in the `nav` tag. You will add navbar links when you are implementing the desktop view. 
- [ ] The hamburger button is currently too large compared to what is given in the Figma design. Kindly reduce the `min-height: 30px;` to be similar with the Figma design.